contents/docs/api/events.md

```diff
@@ -4,10 +4,11 @@ sidebar: Docs
 showTitle: true
 ---
 
+<span class='note-block'>For instructions on how to authenticate to use this endpoint, see [API Overview](/docs/api/api).</span><br />
 
 In PostHog, there is an API endpoint available to see all events in your system and filter them.
 
-**Important:** While you can technically create new events with this API, it's much easier to [use the dedicated API](/docs/integrations/api) or [any of our libraries](/docs/integrations/api) for that.
+**Important:** While you can technically create new events with this API, it is simpler to [use the dedicated event capture API](/docs/api/post-only-endpoints) or [any of our libraries](/docs/integrations) for that.
```

Comment:
That's actually a bit weird. I don't think we should be allowing creation of events through other means then the optimized `/capture`, that also includes some more logic.

Reply:
This is from the old Docs - I just updated the writing. Happy to take this note out if you wish

Reply:
That's more about the server than the docs, since it does appear that the docs are right here. It's just that it's not a good idea for the server to do this. I'll disable that behavior later.